### PR TITLE
Change method of parsing the material type from material name

### PR DESCRIPTION
**Pull request by pklatka**
This PR:
1. Adds supported materials note to the README.md
2. Changes the way how material types were being detected - now we check if the material name starts with pre-defined material name which is available on our side - expected name of material should be, for example, `metal_red`, `metal_yellow`, etc.

Depends on #14

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,22 +4,21 @@
 
 Run before building dependencies from source:
 
-# Ares
+### Ares
```
Comment (pklatka) [inline README.md:7]: Minor changes to the README.md file, should be a separate PR though

**Review by bartektempka — APPROVED** [review]
